examples/google-auth/src/api/labels.js

```diff
@@ -1,67 +1,67 @@
-const url = require("url");
-const { google } = require("googleapis");
+const url = require("url")
+const { google } = require("googleapis")
 
-const clientId = process.env.GOOGLE_CLIENT_ID;
-const clientSecret = process.env.GOOGLE_CLIENT_SECRET;
+const clientId = process.env.GOOGLE_CLIENT_ID
+const clientSecret = process.env.COOGLE_CLIENT_SECRET
```

Comment:
`GOOGLE_CLIENT_SECRET`?

Reply:
Hmm I fixed that already. not sure why it came back.